google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -676,6 +676,88 @@ def signed_url path, method: nil, expires: nil, content_type: nil,
         end
 
         ##
+        # Generate a PostObject that includes the fields and url to
+        # upload objects via html forms.
+        #
+        # Generating a PostObject requires service account credentials,
+        # either by connecting with a service account when calling
+        # {Google::Cloud.storage}, or by passing in the service account
+        # `issuer` and `signing_key` values. Although the private key can
+        # be passed as a string for convenience, creating and storing
+        # an instance of # `OpenSSL::PKey::RSA` is more efficient
+        # when making multiple calls to `post_object`.
+        #
+        # A {SignedUrlUnavailable} is raised if the service account credentials
+        # are missing. Service account credentials are acquired by following the
+        # steps in [Service Account Authentication](
+        # https://cloud.google.com/storage/docs/authentication#service_accounts).
+        #
+        # @see https://cloud.google.com/storage/docs/xml-api/post-object
+        #
+        # @param [String] path Path to of the file in Google Cloud Storage.
+        # @param [Hash] policy When provider, the PostObject fields will
+        #   include a Signature based on the JSON representation of this Hash
+        #   and the same policy in Base64 format.
```

Comment:
I'm thinking it's better to err on the side of too much description with this param. What about (changes in bold):

**The security policy that describes what can and cannot be uploaded in the form.** When **provided**, the PostObject fields will include a Signature based on the JSON representation of this Hash  and the same policy in Base64 format. **If you do not provide a security policy, requests are considered to be anonymous and will only work with buckets that have granted `WRITE` or `FULL_CONTROL` permission to anonymous users.**